cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -64,7 +64,15 @@
     (is (= (reformat-string "(t/defn foo [x]\n(+ x 1))")
            "(t/defn foo [x]\n  (+ x 1))"))
     (is (= (reformat-string "(t/defrecord Foo [x]\nCloseable\n(close [_]\n(prn x)))")
-           "(t/defrecord Foo [x]\n  Closeable\n  (close [_]\n    (prn x)))")))
+           "(t/defrecord Foo [x]\n  Closeable\n  (close [_]\n    (prn x)))"))
+    (is (= (reformat-string "(ns example\n(:require [thing.core :as t]))\n\n(t/defn foo [x]\n(+ x 1))\n\n(defn foo [x]\n(+ x 1))"
```

Comment:
Can you make sure that the lines do not exceed 80 characters?

Reply:
Cribbed from [this PR](https://github.com/weavejester/cljfmt/pull/88/files#r92662161) for how to do that, seems like a good fix for the issue :)